.github/workflows/build.yml

```diff
@@ -19,61 +18,65 @@ concurrency:
 jobs:
   test:
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
 
     timeout-minutes: 10
 
     steps:
-      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
+      - name: Checkout repository
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2.4.0
         with:
           fetch-depth: 2
 
-      - name: Set up Node.js ${{ env.NODE_VERSION }}
+      - name: Set up Node.js
         uses: actions/setup-node@04c56d2f954f1e4c69436aa54cfef261a018f458 # renovate: tag=v2.5.0
         with:
-          node-version: ${{ env.NODE_VERSION }}
+          node-version-file: .nvmrc
           cache: npm
 
       - name: Init platform
         run: |
           git config --global core.autocrlf false
           git config --global core.symlinks true
+          git config --global user.email 'renovate@whitesourcesoftware.com'
+          git config --global user.name  'Renovate Bot'
```

Comment:
Where is this used? Do we push git changes?

Reply:
I added for consistency with [renovatebot/renovate](https://github.com/renovatebot/renovate), does it make Git changes? If you think it is unnecessary, I can remove the whole step?

Reply:
so we can remove that, it's required in main repo for `semantic-release`